layouts/partials/navbar.html

```diff
@@ -104,39 +104,30 @@
-          {{ partial "docs/navbar-section.html" ( dict "ctx" . "sectionTitle" "Installing" "section" $spireInstalling "pageUrl" $pageUrl "version" $version ) }}
-          {{ partial "docs/navbar-section.html" ( dict "ctx" . "sectionTitle" "Using" "section" $spireUsing "pageUrl" $pageUrl "version" $version ) }}
-          {{ partial "docs/navbar-section.html" ( dict "ctx" . "sectionTitle" "Developing" "section" $spireDeveloping "pageUrl" $pageUrl "version" $version ) }}
+          {{ partial "docs/navbar-section.html" ( dict "ctx" . "sectionTitle" "About" "section" $spireAbout "pageUrl" $pageUrl "version" $version ) }}
+          {{ partial "docs/navbar-section.html" ( dict "ctx" . "sectionTitle" "Try"  "section" $spireTry "pageUrl" $pageUrl "version" $version ) }}
+          {{ partial "docs/navbar-section.html" ( dict "ctx" . "sectionTitle" "Plan" "section" $spirePlanning "pageUrl" $pageUrl "version" $version ) }}
+          {{ partial "docs/navbar-section.html" ( dict "ctx" . "sectionTitle" "Deploy" "section" $spireDeploying "pageUrl" $pageUrl "version" $version ) }}
+          {{ partial "docs/navbar-section.html" ( dict "ctx" . "sectionTitle" "Maintain" "section" $spireMaintenance "pageUrl" $pageUrl "version" $version ) }}
           
           <span class="navbar-item is-size-3 has-text-weight-light is-uppercase">
-            SPIRE Integrations
+            Specific Integrations
           </span>
           
-          {{ partial "docs/navbar-section.html" ( dict "ctx" . "sectionTitle" "Overview" "section" $spireInt "pageUrl" $pageUrl "version" $version ) }}
-          {{ partial "docs/navbar-section.html" ( dict "ctx" . "sectionTitle" "OIDC Federation" "section" $spireOIDC "pageUrl" $pageUrl "version" $version ) }}
-          {{ partial "docs/navbar-section.html" ( dict "ctx" . "sectionTitle" "SPIRE and Envoy" "section" $spireEnvoy "pageUrl" $pageUrl "version" $version ) }}
-          {{ partial "docs/navbar-section.html" ( dict "ctx" . "sectionTitle" "SPIFFE Libraries" "section" $spireLibraries "pageUrl" $pageUrl "version" $version ) }}
-          
-          <span class="navbar-item is-size-3 has-text-weight-light is-uppercase">
-            Reference
-          </span>
-          
-          <a class="navbar-item" href="https://docs.google.com/document/d/1RZnBfj8I5xs8Yi_BPEKBRp0K3UnIJYTDg_31rfTt4j8/" target="_blank">
-            Design Document: SPIFFE Reference Implementation (SRI)
-          </a>
-    
+          {{ partial "docs/navbar-section.html" ( dict "ctx" . "sectionTitle" "SPIRE Architecture" "section" $spireArchitecture "pageUrl" $pageUrl "version" $version ) }}
+          {{ partial "docs/navbar-section.html" ( dict "ctx" . "sectionTitle" "Microservices Communication" "section" $spireMicroservices "pageUrl" $pageUrl "version" $version ) }}
+          {{ partial "docs/navbar-section.html" ( dict "ctx" . "sectionTitle" "Keyless Authentication" "section" $spireKeyless "pageUrl" $pageUrl "version" $version ) }}
```

Comment:
It would be great to align the language, order and scope of these three headings to the three buckets of use cases outlined on the home page.